apps/music/js/db.js

```diff
@@ -124,7 +124,11 @@ function initDB() {
     App.showCurrentView(function() {
       reparsingMetadata = false;
       // Hide the  spinner once we've displayed the initial screen
-      document.getElementById('spinner-overlay').classList.add('hidden');
+      // The setTimeout is a workaround for Bug 1166500
+      document.getElementById('spinner').classList.add('hidden');
+      setTimeout(function() {
+        document.getElementById('spinner-overlay').classList.add('hidden');
+      }, 100);
```

Comment:
A note: in my patch for bug 1147173, this code got moved to apps/music/js/music.js: https://github.com/mozilla-b2g/gaia/pull/30112/files#diff-9f40338696745f94c2610d539384f58aR175